packages/api-request/src/index.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import jQuery from 'jquery';
+
+const wpApiSettings = window.wpApiSettings;
```

Comment:
Would there be value in having a package that's dedicated to all options/settings etc passed through from the server?  I'm thinking then you'd just do something like:

```js
import { API_SETTINGS } from '@wordpress/settings';
```

Reply:
This is a leftover, I killed this global :)

Aside, I don't think a settings package would be a good thing, these settings are server-side data. An endpoint for these would be good, but until then. Providing config using `wp_add_inline_script` calls is the way to go to keep the packages generic.

Reply:
OOh you're reviewing the first commit only ;)

Reply:
lol ya I jumped on a bit early ha!